package.json

```diff
@@ -12,7 +12,8 @@
     "url": "https://github.com/cypress-io/cypress-realworld-app/issues"
   },
   "engines": {
-    "node": ">=12.22.0"
+    "node": "^14.0.0 || ^16.0.0 || >=18.0.0",
```

Comment:
Node.js 14 entered [end-of-life](https://github.com/nodejs/release#release-schedule) on Apr 30, 2023, so it should not be listed.

The project is not compatible with Node.js 18, so this should be removed. See https://github.com/cypress-io/cypress-realworld-app/issues/1285#issuecomment-1517859635

The recommended version of Node.js is 16.16.0 according to https://github.com/cypress-io/cypress-realworld-app/blob/develop/.node-version

Reply:
done